src/utils.js

```diff
@@ -453,29 +453,19 @@ exports.insertElement = function(elm, doc, target) {
   } catch (e) {}
 };
 
-exports.insertPixel = function (url) {
+exports.triggerPixel = function (url) {
   const img = new Image();
-  img.id = _getUniqueIdentifierStr();
   img.src = url;
```

Comment:
I think it was @dbemiller who made the suggestion to not add elements to the DOM which we don't need to. I believe my research suggested that simply creating the JS image object with a src was enough for all browsers.